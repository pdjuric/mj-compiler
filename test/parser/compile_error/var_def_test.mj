program main_test
char c;
{
    void foo(int a)
        int a; 
    {

    }
    void main() 
        int d; 
        int b, b;
        int c;
        char d, e;
    {}
}