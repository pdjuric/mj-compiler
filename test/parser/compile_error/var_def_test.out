ERROR 17:13:29,560 - line   5 - Identifier a already defined in this scope.
ERROR 17:13:29,560 - line  11 - Identifier b already defined in this scope.
ERROR 17:13:29,560 - line  13 - Identifier d already defined in this scope.